src/PrestaShopBundle/Resources/config/services/adapter/alias.yml

```
@@ -0,0 +1,30 @@
+services:
+  _defaults:
+    autowire: true
+    public: false
+
+  PrestaShop\PrestaShop\Adapter\Alias\Validate\AliasValidator:
+    autowire: true
+    public: false
```

already inherts defaults
```suggestion
  PrestaShop\PrestaShop\Adapter\Alias\Validate\AliasValidator: ~
```